lib/rbkit.rb

```diff
@@ -106,15 +31,15 @@ def self.start_server(pub_port: DEFAULT_PUB_PORT, request_port: DEFAULT_REQ_PORT
   # profiling is not feasible.
   def self.start_profiling(pub_port: DEFAULT_PUB_PORT, request_port: DEFAULT_REQ_PORT,
```

Comment:
i have a question, not entirely related to this PR.
why does `start_profiling` return an instance of server? what is the use of returning an instance of server?